bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/model/InverterData.java

```diff
@@ -82,6 +84,37 @@ default short getBatteryTemperature() {
         return Short.MIN_VALUE;
     }
 
+    default short getInverterTemperature1() {
+        return Short.MIN_VALUE;
+    }
+
+    default short getInverterTemperature2() {
+        return Short.MIN_VALUE;
+    }
+
+    default short getInverterWorkModeCode() {
+        return Short.MIN_VALUE;
+    }
+
+    @Nullable
+    default String getInverterWorkMode() {
```

Comment:
I just checked the inverter status values in the Cloud API document. From what I can see there are codes 100-113 and the first 6 are just like you parsed them, i.e. 100 is Wait, 101 is Checking. Maybe we could assume that the other codes are the same as here but without the first 1 at the beginning. Could be a nice subsequent feature request...